cmake/Modules/Findblis.cmake

```diff
@@ -0,0 +1,42 @@
+SET(BLIS_INCLUDE_SEARCH_PATHS
+	~/amd-blis-mt/include/blis
```

Comment:
Still referencing a home directory with an AMD specific name

Reply:
These changes are addressed in our recent 2 commits. Kindly refer to latest files. We have removed the AMD specific names as per your suggestion.